docs/userguide.md

```diff
@@ -174,6 +175,13 @@ Status:
     Status:                True
     Type:                  Ready
 ```
+### Unjoin Clusters
+
+If required, federation allows you to unjoin clusters using `kubefed2` tool.
+
+```bash
+kubefed unjoin gondor --host-cluster-context=rivendell
```

Comment:
could you use the same cluster name in `join` case above? It makes consistency to users for reading.